apps/docs/src/app/core/component-docs/card/examples/card-kpi-example.component.html

```diff
@@ -7,7 +7,7 @@
         <h2 fd-card-title>Failure Breakdown</h2>
 
         <fd-card-kpi-header>
-            <strong fd-card-kpi-value fd-object-status status="positive">42.2</strong>
+            <strong fd-card-kpi-value class="fd-numeric-content__kpi--positive">42.2</strong>
```

Comment:
Why do you introduce `class="fd-numeric-content__kpi--positive"`? We shouldn't have fundamental classes exposed like this, that's why we create directives

Reply:
i created input property in `card-kpi-value.directive.ts` to assign class name.